src/front/glsl/ast.rs

```diff
@@ -301,10 +301,12 @@ impl ExpressionRule {
 #[derive(Debug)]
 pub enum TypeQualifier {
     StorageQualifier(StorageQualifier),
+    Interpolation(Interpolation),
     ResourceBinding(ResourceBinding),
     Binding(Binding),
-    Interpolation(Interpolation),
     Sampling(Sampling),
+    Std140,
```

Comment:
We should name this to something like `Layout(StructLayout)` and have
```
enum StructLayout {
  Std140
  // These can be added later
  Std430
  Shared
  Packed
}
```

Reply:
we already have a type with this name
```rust
#[derive(Debug, Clone)]
pub enum StructLayout {
    Binding(Binding),
    Resource(ResourceBinding),
    PushConstant,
}
```